scipy/stats/tests/test_morestats.py

```diff
@@ -523,6 +524,93 @@ def test_result_attributes(self):
         attributes = ('statistic', 'pvalue')
         check_named_results(res, attributes)
 
+    def test_bad_alternative(self):
+        # invalid value for alternative must raise a ValueError
+        x1 = [1, 2, 3, 4]
+        x2 = [5, 6, 7, 8]
+        match = "alternative must be 'two-sided'..."
```

Comment:
```suggestion
        match = "'alternative' must be 'two-sided'"
```

Reply:
My bad! Copied from another function which did not have the `'...'`.

Reply:
No worries, @tupui!

Reply:
It works either way, of course, just good to indicate that it is not the full message if you can spare the characters. And yes, I like the backticks to surround code.